### DISCUSSION TITLE
How to disable pnpm's virtual store

### DISCUSSION THREAD

**Discussion by maticzav** [pnpm/pnpm#4204] [Q&A] [answered]
I've been trying to get Expo working with pnpm and one way or the other I can't seem to figure it out. 

I've seen the posts (bugs) that explain how to get it working, but that seems overly complicated. I say that because if I don't include my workspace package that holds the app in `pnpm-workspace.yml` and just use `yarn` there it works great. My reasoning is that the problem comes from `pnpm` optimising on virtual store and clever linking.

We use all of pnpm's wonderful features in the remainder part of the project, but they make it really difficult to get our app working. Is there an "easy" way to "just" install dependencies of the package like yarn/npm install of a separate project would do it (i.e. no symlinking, no workspace hoisting)?

Thanks 🙂

**Comment by zkochan** [comment, marked as answer]
In the latest pnpm version we released a feature that you are asking for: `node-linker=hoisted`

https://github.com/pnpm/pnpm/releases/tag/v6.25.0
